converter/fluka/input.py

```diff
@@ -1,14 +1,20 @@
-from dataclasses import dataclass
-from converter.fluka.card import Card
+from dataclasses import dataclass, field
+from converter.fluka.cards.card import Card
+from converter.fluka.cards.figure_card import FiguresCard
+from converter.fluka.cards.region_card import RegionsCard
+from converter.solid_figures import SolidFigure
 
 
 @dataclass
 class Input:
-    """Class mapping of the fluka input file."""
+    """Class mapping of the Fluka input file."""
 
     energy_GeV: float = 0.15  # GeV FLUKA specific
     number_of_particles: int = 10000
```

Comment:
```suggestion
    energy_GeV: float = 0.07  # GeV FLUKA specific
    number_of_particles: int = 10000
```

Lets take change hardcoded value to 70 MeV, to be more consistent with documentation https://yaptide.github.io/docs/settings/